packages/backend/src/graphql/queries/test-connection.ts

```diff
@@ -10,20 +11,33 @@ const testConnection: QueryResolvers['testConnection'] = async (
   context,
 ) => {
   let connection = await context.currentUser
-    .$relatedQuery('connections')
+    .withAccessible({
+      type: 'connection',
+      requiredRole: 'viewer',
+    })
```

Comment:
somehow this is preventing me from adding a new excel connection if it doesnt exists